cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -249,8 +254,30 @@ public static EntityTopicOperator fromCrd(Kafka kafkaAssembly) {
             createVolumeMount(EntityOperator.TLS_SIDECAR_CA_CERTS_VOLUME_NAME, EntityOperator.TLS_SIDECAR_CA_CERTS_VOLUME_MOUNT));
     }
 
-    public RoleBindingOperator.RoleBinding generateRoleBinding(String namespace) {
-        return new RoleBindingOperator.RoleBinding(roleBindingName(cluster), EntityOperator.EO_CLUSTER_ROLE_NAME,
-                namespace, EntityOperator.entityOperatorServiceAccountName(cluster), createOwnerReference());
+    public KubernetesRoleBinding generateRoleBinding(String namespace) {
+        KubernetesSubject ks = new KubernetesSubjectBuilder()
+                .withKind("ServiceAccount")
+                .withName(EntityOperator.entityOperatorServiceAccountName(cluster))
+                .withNamespace(namespace)
+                .build();
+
+        KubernetesRoleRef roleRef = new KubernetesRoleRefBuilder()
+                .withName(EntityOperator.EO_CLUSTER_ROLE_NAME)
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("ClusterRole")
+                .build();
+
+        KubernetesRoleBinding rb = new KubernetesRoleBindingBuilder()
+                .withNewMetadata()
+                .withName(roleBindingName(cluster))
+                .withNamespace(namespace)
+                .withOwnerReferences(createOwnerReference())
+                .withLabels(labels.toMap())
+                .endMetadata()
+                .withRoleRef(roleRef)
+                .withSubjects(Collections.singletonList(ks))
```

Comment:
static import for common functions like `singletonList()`, `emptyMap()` where there can be little doubt about what method is being called.